File: driver-service/src/test/java/com/java/akdev/driverservice/util/TestSetUps.java
package com.java.akdev.driverservice.util;

import com.java.akdev.driverservice.dto.DriverCreateDto;
import com.java.akdev.commonmodels.dto.UserResponse;
import com.java.akdev.driverservice.entity.Driver;
import com.java.akdev.driverservice.enumeration.Order;
import com.java.akdev.driverservice.enumeration.SortField;
import lombok.experimental.UtilityClass;

import java.util.UUID;

@UtilityClass
public class TestSetUps {

    public static UUID id = UUID.fromString("4ebba608-6315-447e-9bf5-4e20da6fb0b0");
    public final static Integer DEFAULT_PAGE = 1;
    public final static Integer DEFAULT_PAGE_SIZE = 10;
    public final static SortField SORT_FIELD = SortField.ID;
    public final static Order ORDER = Order.ASC;
    private final static String NAME = "Anton";
    private final static String LAST_NAME = "Kazlouski";
    private final static String NEW_EMAIL = "agusha@gmail.com";
    private final static String EMAIL = "pochta@gmail.com";
    private final static String PHONE_NUMBER = "1234567890";
    private final static String NEW_PHONE_NUMBER = "1234567";
    private final static String PASSWORD = "1234";

    public static Driver getDriver() {
        return Driver.builder()
                .id(id)
                .firstName(NAME)
                .lastName(LAST_NAME)
                .username(EMAIL)
                .phoneNumber(PHONE_NUMBER)
                .password(PASSWORD)
                .build();
    }

    public static DriverCreateDto getCreateDto() {
        return new DriverCreateDto(NAME, LAST_NAME, NEW_EMAIL, PASSWORD, PHONE_NUMBER);
    }

    public static Driver getUpdateDriver() {
        return Driver.builder()
                .id(id)
                .firstName(NAME)
                .lastName(LAST_NAME)
                .username(NEW_EMAIL)
                .phoneNumber(NEW_PHONE_NUMBER)
                .password(PASSWORD)
                .build();
    }

    public static UserResponse getUpdateDto() {
        return new UserResponse(NAME, LAST_NAME, NEW_EMAIL);
    }

    public static UserResponse getReadDto() {
        return new UserResponse(NAME, LAST_NAME, EMAIL);
    }
}

File: passenger-service/src/test/java/com/java/akdev/passengerservice/util/TestSetUps.kt
package com.java.akdev.passengerservice.util

import com.java.akdev.passengerservice.dto.PassengerCreateDto
import com.java.akdev.passengerservice.dto.PassengerReadDto
import com.java.akdev.passengerservice.entity.Passenger
import lombok.experimental.UtilityClass
import java.util.*

@UtilityClass
object TestSetUps {

    val id: UUID = UUID.randomUUID()
    private const val NAME: String = "Anton"
    private const val LAST_NAME: String = "Kazlouski"
    private const val EMAIL: String = "anton@gmail.com"
    private const val PHONE_NUMBER: String = "1234567890"
    private const val PASSWORD: String = "123"

    fun getPassenger(): Passenger {
        return Passenger.builder()
            .id(id)
            .firstName(NAME)
            .lastName(LAST_NAME)
            .username(EMAIL)
            .phoneNumber(PHONE_NUMBER)
            .password(PASSWORD)
            .build()
    }

    fun getCreateDto(): PassengerCreateDto {
        return PassengerCreateDto(NAME, LAST_NAME, EMAIL, PASSWORD, PHONE_NUMBER)
    }

    fun getReadDto(): PassengerReadDto {
        return PassengerReadDto(NAME, LAST_NAME, EMAIL)
    }

}